sipa/babel.py

```diff
@@ -1,32 +1,12 @@
 # -*- coding: utf-8 -*-
-from babel.core import Locale, UnknownLocaleError
+from babel.core import Locale
 
 from flask import request
-from flask_babel import Babel, get_locale
+from flask_babel import Babel
 
 babel = Babel()
 
 
-def locale_preferences():
```

Comment:
Please test your code before pushing. Although you replaced the usage of `locale_preferences` by `negotiate_locale`, you forgot to delete the import in [flatpages.py#L11](https://github.com/agdsn/sipa/blob/4938ce981eeecb35678d55438b1bfaffd99b12ea/sipa/flatpages.py#L11), which is also the rason for the CI failure.

Reply:
Because this is a one-liner I did not want to add a commit, just amending should be fine.

Reply:
It's fixed.